cloud/src/common/metric.cpp

```diff
@@ -228,10 +230,91 @@ static void export_fdb_status_details(const std::string& status_str) {
     }
 }
 
+void get_partition_boundaries_count(std::vector<std::string>& partition_boundaries,
```

Comment:
add comments for this function

Reply:
what is in boundaries 
what is the key in the map what is the value.

better put an example in the comment what the boundaries and result map look like